sample-service/README.md

```diff
@@ -2,7 +2,7 @@ The sample service is a small HTTP API which is used for testing various integra
 
 This repo contains a simple Helm chart for deploying the sample service on Kubernetes.
 
-The code for the sample service can be found at [RoadiqHQ/sample-service](https://github.com/roadiehq/sample-service).
+The code for the sample service can be found at [RoadieHQ/sample-service](https://github.com/roadiehq/sample-service).
```

Comment:
Will we be adding the broker in the next commit?

I think we need the following for the broker part:

```
{{ if .Values.broker.enabled }}
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: broker-pod
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: broker-pod-role-binding
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: {{ .Values.clusterRole.name }}
subjects:
  - kind: ServiceAccount
    name: broker-pod
    namespace: default
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: broker-client
  labels:
    app: broker
spec:
  replicas: 1
  selector:
    matchLabels:
      app: broker
  template:
    metadata:
      labels:
        app: broker
    spec:
      serviceAccountName: broker-pod
      containers:
      - name: broker
        image: roadiehq/broker:kubernetes
        imagePullPolicy: Always
        env:
          - name: BROKER_TOKEN
            value: {{ .Values.brokerToken }}
          - name: BROKER_SERVER_URL
            value: https://{{ .Values.tenantName }}.broker.roadie.so
        ports:
        - containerPort: 8000
{{ end }}
```

Reply:
Yes we discussed already - want to get this out and then look at broker and test it etc..